packages/image_picker/image_picker_platform_interface/lib/src/types/image_picker_options.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:image_picker_platform_interface/src/types/types.dart';
+
+/// Specifies options for picking a single image from the device's camera or gallery.
+class ImagePickerOptions {
+  /// Creates an instance with the given [maxHeight], [maxWidth], [imageQuality],
+  /// [referredCameraDevice] and [requestFullMetadata]. Any of the params may be null.
+  const ImagePickerOptions({
+    this.maxHeight,
+    this.maxWidth,
+    this.imageQuality,
+    this.preferredCameraDevice = CameraDevice.rear,
+    this.requestFullMetadata = true,
+  });
+
+  /// If specified, the image will be at most `maxWidth` wide. Otherwise,
+  /// the image will be returned at its original width.
+  final double? maxWidth;
+
+  /// If specified, the image will be at most`maxHeight` tall.
+  /// Otherwise the image will be returned at its original height.
+  final double? maxHeight;
+
+  /// Modifies the quality of the image, ranging from 0-100 where 100 is the original/max
+  /// quality. If `imageQuality` is null, the image with the originalquality will
```

Comment:
"If null, the image will be returned at its original quality."

Reply:
Also, lets put a blank comment line after the first sentence here, per https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph